src/LiveDevelopment/transports/node/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "brackets-livedev-server",
+  "dependencies": {
+    "ws": "~0.4.31"
```

Comment:
I think you can put it on the root package.json and update the shrinkwrap file.
Right now, I don't think this works.